packages/2019-disaster-game/src/components/atoms/Orb.js

```diff
@@ -75,21 +75,16 @@ export default class Orb extends PureComponent {
   }
 
   incrementGauge = () => {
-    const timer = setTimeout(() => {
-      const { isActive } = this.state;
-      if (isActive) {
-        this.setState({ isComplete: true });
-        clearTimeout(timer);
-      } else {
-        this.setState({ isActive: false });
-        clearTimeout(timer);
-      }
-    }, pressSuccessDuration * 1000);
+    const { isActive } = this.state;
+    if (isActive) {
+      this.setState({ isComplete: true });
+    }
```

Comment:
It looks like this changes the way the game works a bit. Now the item is captured immediately instead of after a second? 

Probably unimportant, but figured I'd bring it up.

Reply:
That's exactly the mechanics we want. Codes a little weird, but it works.